pkg/driver_sx126x/Makefile.dep

```diff
@@ -0,0 +1,8 @@
+# module dependencies
+USEMODULE += fmt
```

Comment:
I think I missed where this module was used

Reply:
This is a leftover from previous debugging, will remove